doc/man7/EVP_MD-BLAKE2.pod

```diff
@@ -0,0 +1,46 @@
+=pod
+
+=head1 NAME
+
+EVP_MD-BLAKE2 - The BLAKE2 EVP_MD implementation
+
+=head1 DESCRIPTION
+
+Support for computing SHA2 digests through the B<EVP_MD> API.
+
+=head2 Identities
```

Comment:
Have you purposely decided to use the term "Identities"? Wouldn't be "Identifiers" be more appropriate?  (multiple occurrences)

Reply:
I've consistently used "Identity" (singular and plural).  We have that pattern already in the EVP_MAC and EVP_KDF descriptions...